### PR TITLE
Restore access to all headers on Chrome 72+

### LINKED ISSUE #2235: webRequest header access changes in Chrome 72
It looks like Privacy Badger is accessing and modifying headers that will not be available by default in Chrome 72 (currently on the dev channel). These include:

Request headers:
- Accept-Language
- Accept-Encoding
- Cookie
- Referer

Response headers:
- Set-Cookie

To access these headers you will need to add 'extraHeaders' in the extraInfoSpec of the relevant webRequest listeners.

See the post on the chromium-extensions Google Group for more info: https://groups.google.com/a/chromium.org/forum/#!topic/chromium-extensions/vYIaeezZwfQ

### PR DESCRIPTION
**Pull request by ghostwords**
Fixes #2235.

### CODE REVIEW

File: src/js/heuristicblocking.js [hunk 1]
```diff
@@ -187,9 +187,9 @@ HeuristicBlocker.prototype = {
     // not-yet-blocked domain for DNT policy.
     // We check heuristically-blocked domains in webrequest.js.
     if (!skip_dnt_check) {
-      window.setTimeout(function () {
+      setTimeout(function () {
         badger.checkForDNTPolicy(tracker_fqdn);
-      }, 10);
+      }, 0);
```
Comment (bcyphers) [inline src/js/heuristicblocking.js]: Sorry if I'm misunderstanding here -- why setTimeout if there's no timeout? Is this just to make the call async?

Comment (ghostwords) [reply src/js/heuristicblocking.js]: Yep: https://johnresig.com/blog/how-javascript-timers-work/

Comment (bcyphers) [reply src/js/heuristicblocking.js]: Sorry if I'm misunderstanding here -- why setTimeout if there's no timeout? Is this just to make the call async?

Comment (bcyphers) [reply src/js/heuristicblocking.js]: Thanks! Mind adding a short comment to that effect?

**Pull request merged**